tutorials/qdrant_langchain_instrumentation_search_and_retrieval_tutorial.ipynb

```diff
@@ -0,0 +1,478 @@
+{
```

Comment:
<p>Explain how they can view in Phoenix, perhaps by printing out the session url</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/2634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>